test_suite/package/test_efs_utils.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from lib import test_lib
+
+
+@pytest.mark.package
+@pytest.mark.run_on(['rhel9.4'])
+class TestsEfsUtils:
+    def test_efs_utils(self, host, instance_data):
+        """
+        NOTE: This test case assumes efs-utils is preinstalled in the AMI image
+        Check basic functionality of EFS utils:
+        - Create a mount point and mount the EFS file system to the RHEL instance.
+        - Create a file on the mount point.
+        - Automate mount on boot.
+        - Checksum the file before reboot.
+        - reboot and checksum that the file is correct.
+        """
+        # will get 'file_system_dns_name' arg form the fixture that gets the output of the Terraform script
+        file_system_dns_name = instance_data['efs_file_system_dns_name']
+        test_file = '/mnt/efs/testfile'
+        with host.sudo():
+            assert host.run_test(
+                f'mkdir /mnt/efs && mount -t efs {file_system_dns_name} /mnt/efs/'
+            ), f'Failed mount {file_system_dns_name} to efs folder'
```

Comment:
```suggestion
            ), f'Failed mount {file_system_dns_name} to efs directory'
```
Linux naming convention is better for us ;)